ogticrd/covid-safe-paths
# Feat: Encrypting coordinates

#### Description:


Encrypting users coordinates to make sure that they are transferring securely with base64 conversion, I had to install a package called base64-react-native due to react native does not compatible with buffer node module.
#### Linked issues:


Feat: [274](https://trello.com/c/khRffpOo/274-as-a-user-i-want-my-cordenades-send-encripted-so-that-i-can-be-sure-no-one-is-tracking-me-2)

#### How to test:


- `git pull && git checkout Feat/274-encypting-coordinates`
- Report yourself as positive covid user and change your location 
- The app should send your location encrypted 

**Note:** Talk to @epolanco to confirm that coordinates has been sent
